docs/source.rst

```diff
@@ -52,20 +52,19 @@ organization you are submitting to.
 |Source Interface with Javascript Disabled|
 
 If this is the first time you're using the Tor browser, it's likely that you
-have JavaScript enabled and that the Security Slider that Tor browser provides
+have JavaScript enabled and that the Security Setting that Tor browser provides
```

Comment:
preexisting nit: "the Tor browser"

Reply:
I will see your `nit` and raise you consistent capitalization throughout the doc :-) (eabb86ab8)